src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -473,60 +443,35 @@ private void OnStateChanged(NetworkPeerState previous)
         {
             this.logger.LogTrace("({0}:'{1}')", nameof(message), message.Message.Command);
 
-            switch (message.Message.Payload)
+            try
```

Comment:
The method `ProcessMessageAsync` should probably not be public

Reply:
true, tx, fixed